### DISCUSSION TITLE
Gigabyt B460M DS3H, No matter what I've tried I have still not been able to flash the moded Bios file.

### DISCUSSION THREAD

**Discussion by qonraad** [terminatorul/NvStrapsReBar#96] [Q&A]
can some one give me a hand flashing the modded bios file. So I have followed from the start and following all the steps I end up at the page "Use Qflash to flash modified BIOS. If Qflash gives an error then see [Flashing Gigabyte while avoiding “Invalid BIOS image” following some of the steps on that page still no luck.

**Comment by Aserius4034** [comment]
Same with my AB350M-Gaming 3 zzzz
